jiant/scripts/postproc/xtreme/README.md

```diff
@@ -0,0 +1,369 @@
+# XTREME Running / Submission Guide
```

Comment:
Is it possible to make this file executable so that the README.md and XTREME E2E test is one and the same?

Reply:
Hmm, as a compromise I could convert the bash segments to separate scripts, and link them instead of putting them in-line. (I would prefer the guide/documentation remain in markdown). We can have an E2E script call all the subscripts at once.

Reply:
(If I'm not wrong, GitHub doesn't currently have an embed-file feature in markdown, unfortunately.)

Reply:
(We'll still have to kick off the final run commands in parallel. But I do agree that the more we can de-dup, the better.)

Reply:
This sounds great!

Reply:
Done.